src/app/services/shared.service.ts

```diff
@@ -45,11 +54,31 @@ export class SharedService {
         this.toastrService.warning(message.description, message.title);
     }
 
+    public set status(value: ConnectivityStatus) {
```

Comment:
Why the getter/setter, if they're trivial and public? I think we don't need these javisms in TS, IMO ;)

Reply:
Besides, seems like it's only used in that `hasRpcError`. Wouldn't it be more simple just testing for the stack trace, as they're always set together?

Reply:
I mostly had in mind to add some kind of connectivity report here afterwards. 
For example if the node stops but you still have the webui loaded in your browser.

However I can unify everything and then if there is the need I can modify.